.github/workflows/ci.yml

```diff
@@ -24,12 +24,16 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        node-version: [10.x, 12.x]
+        node-version: [10.x, 12.x, 13.x]
         exclude:
           - os: macos-latest
             node-version: 10.x
+          - os: macos-latest
+            node-version: 13.x
           - os: windows-latest
             node-version: 10.x
+          - os: windows-latest
+            node-version: 13.x
```

Comment:
why node 13.x ?

Reply:
This is an exclude - mac and windows are only tested on node 12

Reply:
sorry, missed that.